ansible/inventory/inventory_rke2_ha.yml

```diff
@@ -0,0 +1,40 @@
+#control_nodes:
+#  hosts:
+#    token_node:
+#      ansible_host:
+#      config_file:
+#    optional_control_node_1:
+#      ansible_host:
+#      config_file:
+#    optional_control_node_2:
+#      ansible_host:
+#      config_file:
+#
+#agent_nodes:
+#  hosts:
+#    optional_agent_1:
+#      ansible_host:
+#      config_file:
+#    optional_agent_2:
+#      ansible_host:
+#      config_file:
+#    optional_agent_3:
+#      ansible_host:
+#      config_file:
+control_nodes:
+  hosts:
+    token_node:
+      ansible_host: 18.117.9.0
+      config_file: /Users/wzya/Desktop/Wojtek/git_repos/splunk-connect-for-syslog/c1.yml
+    optional_control_node_1:
+      ansible_host: 13.59.236.185
```

Comment:
Please remove IP and personal uid signatures.

Reply:
Done